sklearn/feature_extraction/_dict_vectorizer.py

```diff
@@ -98,6 +119,27 @@ def __init__(self, *, dtype=np.float64, separator="=", sparse=True,
         self.sparse = sparse
         self.sort = sort
 
+    def _add_iterable_element(self, f, v, feature_names, vocab, fitting=True,
+                              transforming=False, indices=None, values=None):
```

Comment:
A function like this is less error prone for callers if either its parameters don't have default values defined, or they are keyword-only.